specification/security/resource-manager/Microsoft.Security/stable/2023-11-15/apiCollections.json

```diff
@@ -0,0 +1,473 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
+    "description": "API spec for Microsoft.Security (Microsoft Defender for Cloud) resource provider",
+    "version": "2023-11-15"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/apiCollections": {
+      "get": {
+        "x-ms-examples": {
+          "Gets a list of API collections within a subscription that have been onboarded to Microsoft Defender for APIs": {
+            "$ref": "./examples/ApiCollections/APICollections_ListBySubscription_example.json"
+          }
+        },
+        "tags": [
+          "D4APICollectionList"
+        ],
+        "description": "Gets a list of API collections within a subscription that have been onboarded to Microsoft Defender for APIs.",
+        "summary": "Gets a list of API collections within a subscription",
+        "operationId": "APICollections_ListBySubscription",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "This indicates a successful response. The response contains the list of API collections.",
+            "schema": {
+              "$ref": "#/definitions/ApiCollectionList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
```

Comment:
>v3 [](http://example.com/codeflow?start=71&length=2)

Best to use a consistent common types version, i.e. v5. Comment applies throughout this PR.

Reply:
Thanks for catching this! I missed this when I authored the file. I've updated it